README.md

```diff
@@ -18,12 +18,14 @@ there is a specialized collection called `JSONCollection` that deals naturally w
 
 #### Add simple-reactivemongo
 
-In your project/Build.scala:
+In your project/build.sbt:
 
 ```scala
+resolvers += Resolver.bintrayRepo("hmrc", "releases")
+
 libraryDependencies ++= Seq(
-  "uk.gov.hmrc" %% "simple-reactivemongo" % "[INSERT_VERSION]",
```

Comment:
the reason it is "[INSERT_VERSION]" was because we had some ask us where version "x.x.x" was